### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by kyleyoungblom** [sundevista/local-quotes#21] [Ideas]
Hello, I'm wondering if there could be a way to source quotes by theme or topic instead of just author.

For example: if I had quotes by Audre Lorde and Simone Weil and Albert Camus that were all related to the same idea, I could search for that idea and be presented with a rotating selection of those quotes.

This would be useful for me as I have many pages dedicated to a specific area of focus, and I would love to bring in quotes relevant to that subject.

**Comment by sundevista** [comment]
I had thoughts about it too. I will start to develop this feature in the nearest available time.

**Comment by sundevista** [comment]
https://trello.com/c/zmQy1XwS/3-develop-a-new-feature-tags
